packages/js-launchdarkly-provider/src/lib/js-launchdarkly-provider.ts

```diff
@@ -1,24 +1,46 @@
 import {
   Context,
+  ContextTransformer,
   FeatureProvider,
   FlagEvaluationOptions,
   FlagTypeError,
   FlagValue,
-  FlagValueParseError
+  FlagValueParseError, ProviderOptions
 } from '@openfeature/openfeature-js';
-import { init, LDClient } from 'launchdarkly-node-server-sdk';
+import { init, LDClient, LDUser } from 'launchdarkly-node-server-sdk';
+
+export interface LaunchDarklyProviderOptions extends ProviderOptions<LDUser> {
+  sdkKey: string;
+}
+
+/**
+ * Transform the context into an object compatible with the Launch Darkly API, an object with a user "key", and other attributes.
+ */
+const DEFAULT_CONTEXT_TRANSFORMER = (context: Context): LDUser => {
```

Comment:
New transformer for LauchDarkly. Standard attributes are not supported (only custom), but when we decide on standard attributes for the openfeature Context schema, we could do that mapping here.